Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -97,10 +97,12 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 x90_wf, x90_der_wf = np.array(drag_gaussian_pulse_waveforms(x90_amp, x90_len, x90_sigma, alpha=0, delta=1))
 # No DRAG when alpha=0, it's just a gaussian.
 
-x270_len = x180_len
-x270_sigma = x270_len / 5
-x270_amp = -x90_amp
-x270_wf, x270_der_wf = np.array(drag_gaussian_pulse_waveforms(x270_amp, x270_len, x270_sigma, alpha=0, delta=1))
+minus_x90_len = x180_len
```

Comment:
I thought that there was an agreement to call it `x270` and not `minus_x90`.
I don't really care what we choose, but:
1. Let's make a decision because you'll change it to `minus_x90` and tomorrow someone would want to change it back.
2. We need it to be consistent amount all of our QCAs